packages/backend/src/tokens/keys.ts

```diff
@@ -202,6 +208,31 @@ async function fetchJWKSFromBAPI(apiUrl: string, key: string, apiVersion: string
   });
 
   if (!response.ok) {
+    const errors = await response.json().then(({ errors }) => {
```

Comment:
🔧  I can see we already use `await`, so let's drop `then` to make it more readable.

Example:
```ts
const json = await response.json();
const errors = buildErrorOrSomething(json?.errors);
...
```

Reply:
@anagstef yes, this makes sense